app/jobs/bulk_actions_job.rb

```diff
@@ -0,0 +1,48 @@
+class BulkActionsJob < ApplicationJob
+  queue_as :medium
+  attr_accessor :conversations
+
+  def perform(account:, params:, method_type:, user:)
+    @conversation_ids = params.delete(:conversation_ids)
+    @labels = params.delete(:labels)
+    @conversations = account.conversations.where(display_id: @conversation_ids)
+    @method_type = method_type
+    Current.user = user
+    bulk_update(params)
+  end
+
+  def bulk_update(params)
+    if @method_type == :delete
+      bulk_delete_conversation(params)
+    else
+      bulk_conversation_update(params)
+    end
+  end
+
+  def bulk_conversation_update(params)
+    params = available_params(params)
+    conversations.each do |conversation|
+      add_labels(conversation)
+      conversation.update(params)
+    end
+  end
+
+  def bulk_delete_conversation(_params)
```

Comment:
let's consider renaming this method since it doesn't reflect the action it performs. 
also, the parameters seem to be unused in this method.